students/kahyee/Session02/series.py

```diff
@@ -0,0 +1,39 @@
+def fibonacci(x):
+	"""Returns the xth term of the fibonacci series"""
+	fib1 = 0
+	fib2 = 1
+	answer = 0
+
+	for i in range(x):
+		answer, fib1, fib2 = fib1, fib2, fib1 + fib2
+	return answer
+
+# print(fibonacci(0))
+# print(fibonacci(3))
+# print(fibonacci(5))
+# print(fibonacci(100))
+
+def lucas(x):
+	"""Returns the xth term of the lucas series"""
+	luc1 = 2
+	luc2 = 1
+	answer = 0
+
+	for i in range(x):
+		answer, luc1, luc2 = luc1, luc2, luc1 + luc2
+	return answer
+
```

Comment:
Lucas numbers are very similar to fibonacci, they just are invalid for different numbers.